.github/workflows/deploy.yml

```diff
@@ -6,6 +6,8 @@ on:
   push:
     branches:
       - main
+    tags: 
```

Comment:
a pattern here might be a good preventative measure for PyPI PEP 440 compatibility, something like: `[0-9]+.[0-9]+.[0-9]+`, only for `pypi` trigger needed, since it’s the only one triggering on tags.